pg-extend/src/pg_alloc.rs

```diff
@@ -31,7 +31,7 @@ impl PgAllocator {
         unsafe { Self::from_raw(pg_sys::CurrentMemoryContext) }
     }
 
-    /// Sets this PgAllocator as the current memory context, and then resets it to the orevious
+    /// Sets this PgAllocator as the current memory context, and then resets it to the previous
     ///  after executing the function.
```

Comment:
The two spaces on this line should be replaced with 1 space.

Reply:
Done